### PR TITLE
Made windows compiler for easy compilation

### PR DESCRIPTION
**Pull request by neoapps-dev**
i made it so you can compile the app easily :moai:

### CODE REVIEW

**Comment by Timoh5709** [comment]
Finally, someone that simplifies this compiling thing for windows ! 👍

**Comment by neoapps-dev** [comment]
> Finally, someone that simplifies this compiling thing for windows ! 👍

a windows user will care about that lol

**Comment by GStudiosX2** [comment]
Rename the file to install-windows.bat for consistency

**Comment by face-hh** [comment]
image

**Comment by GStudiosX2** [comment]
> image

Try running it in a CMD prompt maybe ? Possible the script doesn't support powershell but I didn't make it so /shrug

**Comment by face-hh** [comment]
I've tried running the bat directly and through cmd, same thing. Haven't tried powershell

**Comment by GStudiosX2** [comment]
> I've tried running the bat directly and through cmd, same thing. Haven't tried powershell

Oh i thought you where opening it in powershell you could try it though

**Comment by face-hh** [comment]
image

**Comment by neoapps-dev** [comment]
woah, the script i added to github is old.. im gonna update it rq

File: README.md [hunk 1]
```diff
@@ -61,19 +61,12 @@ Then you could just launch it using `webx` in your terminal.
...
-2. Download GNU target: `rustup toolchain install stable-gnu && rustup default stable-gnu`
-3. Download [MSYS32](https://www.msys2.org/)
-4. Open MSYS32 MINGW32
-5. Run: `pacman -Syu` just in case.
-6. Run `pacman -S mingw-w64-x86_64-toolchain base-devel mingw-w64-x86_64-gtk4 mingw-w64-x86_64-gettext mingw-w64-x86_64-libxml2 mingw-w64-x86_64-librsvg mingw-w64-x86_64-pkgconf mingw-w64-x86_64-gcc mingw-w64-x86_64-libadwaita mingw-w64-x86_64-lua`
-7. Go to `Settings` -> `Search` and open `Advanced system settings` -> Click on `Environment variables` (or just search "path")
-8. Select `Path` -> Click on `Edit` -> Add the following three entries: `C:\msys64\mingw64\include`, `C:\msys64\mingw64\bin`, and `C:\msys64\mingw64\lib`.
-9. Open a terminal in the folder with `napture/`, run `cargo run`.
+1. Download this repo as a zip and extract it.
+2. Run `windows-compiler.bat` by @neoapps-dev
```
Comment (QuinceTart10) [inline README.md:68]: I think it's pretty stupid to credit yourself like this. Just remove it please.
```suggestion
2. Run `windows-compiler.bat`
```

Comment (neoapps-dev) [reply README.md:68]: bro, can't i be credited for my work?

Comment (QuinceTart10) [reply README.md:68]: I just think it looks stupid, plus your name is forever visible in the commit history of that file.

Comment (neoapps-dev) [reply README.md:68]: alright D:

Comment (neoapps-dev) [reply README.md]: done

**Comment by SticksDev** [comment]
batch is old and the outdated standard for Windows. I'm planning to re-write this in PowerShell which is more modern and allows for some other things that batch cannot do, e.g process waiting.

**Comment by neoapps-dev** [comment]
> batch is old and the outdated standard for Windows. I'm planning to re-write this in PowerShell which is more modern and allows for some other things that batch cannot do, e.g process waiting.

you could use powershell inside of batch 🗿 and i did that of course.

**Comment by face-hh** [comment]
Moved to #152

**Pull request closed without merging**